### PR TITLE
Fixed pinpoint problem issue on the map

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/contact-us/_offices.js [hunk 1]
```diff
@@ -126,11 +126,11 @@ export const Offices = () => {
                             <MapLink
                                 rel="noopener noreferrer"
                                 target="_blank"
-                                href="https://maps.google.com/?q=Binary+(Europe)+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
+                                href="https://maps.google.com/?q=Binary+Europe+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
                             >
                                 <Image
                                     img_name="map-malta.png"
-                                    alt={localize('Malta Office')}
+                                    alt={localize('Maltaa Office')}
```
Comment (mustofa-binary) [inline src/pages/contact-us/_offices.js]: typo?

File: src/pages/contact-us/_offices.js [hunk 2]
```diff
@@ -149,7 +149,7 @@ export const Offices = () => {
                                     target="_blank"
                                     color="black-3"
                                     size="var(--text-size-s)"
-                                    href="https://maps.google.com/?q=Binary+(Europe)+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
+                                    href="https://maps.google.com/?q=Binary+Europe+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
```
Comment (mustofa-binary) [inline src/pages/contact-us/_offices.js]: why does this make any difference?

File: src/pages/contact-us/_offices.js (hunk 1 above)
Comment (sara-fs) [reply src/pages/contact-us/_offices.js]: Ah, no it was a test. Forgot to undo.

File: src/pages/contact-us/_offices.js (hunk 2 above)
Comment (sara-fs) [reply src/pages/contact-us/_offices.js]: Really don't know!!! but it didn't work like that!

**Pull request merged**